### PR TITLE
[SOUNDS-922] Add icons and colours for Radio Cymru 2

### PR DESCRIPTION
**Pull request by craigharvi3**
## Jira
https://jira.dev.bbc.co.uk/browse/SOUNDS-922

## Description
Add colours and icons for radio cymru 2

### CODE REVIEW

File: assets/colours/networks.json [hunk 1]
```diff
@@ -344,6 +344,21 @@
                 }
             }
         },
+        "bbc_radio_cymru_2": {
+            "default": {
+                "colour1": "#1db09e",
+                "colour2": "#1db09e",
+                "colour3": "#1db09e",
```
Comment (davidrhyswhite) [inline assets/colours/networks.json]: All 3 colours are the same? 🤔

Comment (craigharvi3) [reply assets/colours/networks.json]: Yep, we've only been given 1 colour lol.

Comment (davidrhyswhite) [reply assets/colours/networks.json]: Also this colour isn't red like the Welsh dragon...? 🙈

Comment (craigharvi3) [reply assets/colours/networks.json]: Yep haha. I think it's just a slight variation of the Cymru one test.bbc.co.uk/radio/play/live:radiocymru

**Comment by craigharvi3** [comment]
Excellent approval @davidrhyswhite 👏

**Pull request merged**